ui/StatusQ/tests/tst_MovableModel.cpp

```diff
@@ -647,9 +647,12 @@ private slots:
 
         model.setSourceModel(sourceModel2);
 
-        QCOMPARE(signalsSpy.count(), 2);
+        QCOMPARE(signalsSpy.count(), 4);
         QCOMPARE(signalsSpy.modelAboutToBeResetSpy.count(), 1);
         QCOMPARE(signalsSpy.modelResetSpy.count(), 1);
+        QCOMPARE(signalsSpy.layoutAboutToBeChangedSpy.count(), 1);
```

Comment:
Why do we need to emit layoutAboutToBeChanged/layoutChanged on reset?

Reply:
Yeah, but ideally model shouldn't send unnecessary signals, so ideally source change should trigger only reset, without layout change. It would require some internal changes to not emit that signal in this case.